### DISCUSSION TITLE
Cannot sign up an account.

### DISCUSSION THREAD

**Discussion by JackyShao01** [simple-login/app#1354] [Self-hosting question]
I followed instructions in Readme.md and tried to self-host the simplelogin app . The simplelogin web can be correctly navigated. However, when i signed up an account, I got the error message "You cannot use this email address as your personal inbox." whatever mail addresses have been used. Please see the attached screenshot.  
<img width="584" alt="Capture" src="https://user-images.githubusercontent.com/13310383/196318147-e16f88d0-b25f-4b37-b320-bdc2693f328a.PNG">
I am using CentOS  EC2 as the host.  
And in sl_app log, it is stated no mx record for these domain. But I do have  a domain which was created by following the readme.md.
<img width="842" alt="Capture2" src="https://user-images.githubusercontent.com/13310383/196320279-179cd6fc-65be-4294-af09-b391c83484c7.PNG">

What is causing this? How should I fix this.  Thank you.

**Comment by nguyenkims** [comment]
Have you tried with a ProtonMail or Gmail address to see if the error goes away? By default SimpleLogin runs a check to make sure the mailbox address isn't an alias.

By the way, I'd recommend not using EC2 to self host SL as by default it blocks the port 25 for sending emails and AWS IP addresses have bad reputation in terms of email delivery.

**Reply by JackyShao01** [reply]
Hi Kim, Thanks for replying. 
Yes, I did try my gmail when creating one simplelogin account. (Please refer to the attached figure.)
<img width="586" alt="Capture" src="https://user-images.githubusercontent.com/13310383/197428043-21ad3c7b-910d-41c7-aacf-f1a5d02c2c34.PNG">
In addition, I checked the security group in the hosted EC2 and no port 25 is blocked.
When registering with my gmail, the app-sl log shows "no mx record for domain gamil.com"
 
<img width="1183" alt="Capture2" src="https://user-images.githubusercontent.com/13310383/197428491-d9e13e4f-19fa-4968-87d2-fddbe8502ee8.PNG">
When the docker containers was span up, I realized some warming messages and debug info as shown below: 
<img width="927" alt="Capture3" src="https://user-images.githubusercontent.com/13310383/197428680-4d286549-a8ce-479d-9611-05159621f5d6.PNG">

It shows "mydomain.com" is already a SL domain. which I dont know what it means. 
Could these cause the issue?

Please help me. 
Thank you in advance, Kim.

**Reply by nguyenkims** [reply]
> "no mx record for domain gamil.com"

I guess there's a typo in gamil.com and you mean gmail.com :). By default SL makes sure that a domain can receive emails by querying its MX record and from the error message, I suspect that this query has been blocked by AWS. If possible, I'd try to self host on another VPS.

> In addition, I checked the security group in the hosted EC2 and no port 25 is blocked.

Even if the port isn't blocked, your outgoing emails have a great chance of being blocked if they are sent from EC2 ...
